api_backend.yaml

```diff
@@ -476,6 +476,28 @@ paths:
           description: Unavailable service
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/support-token":
+    x-swagger-router-controller: SupportController
```

Comment:
Maybe we can use a generic url instead `support-token`, for example `/token`
Can we use the existing `AuthenticationController`?

Reply:
Endpoint changed :). At the moment, I'd rather keep `SupportController` for other needs if necessary :)